### PR TITLE
Rename Deckard to Stelline.

### PR DESCRIPTION
**Pull request by ximon18**
Maybe we also need a README noting the history of this testing approach?

### CODE REVIEW

**Comment by partim** [comment]
Some introductory comments in `test/net/stelline/mod.rs` would probably be nice.

Would it make sense to lift this from `test` to `src`?

**Review by partim — APPROVED** [review]

**Comment by ximon18** [comment]
> Would it make sense to lift this from `test` to `src`?

Maybe wait with that until we maybe move it all to src so that SystemTime can be mocked via `#[if(cfg)]` as needed by my server side PR #274.

**Comment by ximon18** [comment]
> Some introductory comments in `test/net/stelline/mod.rs` would probably be nice.

As the creator and person with best knowledge of the history and context I hope that is someting @Philip-NLnetLabs can add.

File: tests/net/stelline/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Stelline
+
+Stelline is a test runner based around DNS message request/response sequences defined in a human readable text file format.
+
+## Etymology and origins
+
+The Stelline replay file format is a variant of the original `.rpl` format used by the [Unbound](https://www.unbound.net/) test tool "testbound".
```
Comment (Philip-NLnetLabs) [inline tests/net/stelline/README.md:7]: testbound is based on (or inspired by) on ldns-testns

Comment (ximon18) [reply tests/net/stelline/README.md]: Is that better?

Comment (Philip-NLnetLabs) [reply tests/net/stelline/README.md]: So it should be 'testbound was originally based on ldns-testns'

**Review by Philip-NLnetLabs — APPROVED** [review]
Looks good to me

File: tests/net/stelline/README.md (hunk 1 above)
Comment (ximon18) [reply tests/net/stelline/README.md]: I made some changes, let me know if it's good now.

**Pull request merged**